mne/annotations.py

```diff
@@ -1576,6 +1577,11 @@ def events_from_annotations(
         they fit within the annotation duration spaced according to
         ``chunk_duration``. As a consequence annotations with duration shorter
         than ``chunk_duration`` will not contribute events.
+    tol : float
+        The tolerance which is used to check if the calculated onsets have the
+        enough duration to the end of the annotation. If the duration from the
+        onset to the end of the annotation is smaller than ``chunk_duration``
+        minus ``tol``, the onset will be discarded.
```

Comment:
can you clarify that this tol parameter is only useful when chunk_duration != None?

Reply:
fixed:
https://github.com/mne-tools/mne-python/pull/12324/commits/8b146a2454f26d7f194d522861e0411677fa8d18